src/dataregistry/schema/schema.yaml

```diff
@@ -288,3 +284,18 @@ dataset:
   move_date:
     type: "DateTime"
     description: "Date the dataset was last moved"
+  location_type:
+    type: "String"
+    choices: ["dataregistry", "external", "dummy"]
+    cli_default: "dataregistry"
+    cli_optional: True
+    nullable: False
+    description: "What is the physical location of the data? 'dataregistry' means the data is located within the `<root_dir>` and managed by the `dataregistry`, 'external' means the data is off-site and not managed by the `dataregistry`, therefore there is only a database entry (in this case a `url` or `contact_email` must be provided during registration) and 'dummy' is a dataset for testing purposes only (only a database entry is created in this case)."
```

Comment:
'external' is not necessarily off-site. Maybe change text to something like

    ... external means the data is not managed by the dataregistry, either because it is off-site or because it is stored outside <root_dir> therefore there is only a database entry ...